internal/fs/foreign_modifications_test.go

```diff
@@ -602,7 +601,7 @@ func (t *ForeignModsTest) ReadBeyondEndOfFile() {
 	AssertEq(contents[contentLen-1], buf[0])
 
 	if err == nil {
-		n, err = f.Read(buf)
+		n, _ = f.Read(buf)
```

Comment:
Instead of this should we put Assert(nil, err) ?

Reply:
I tried doing that. Seems like these errors have been deliberately ignored. Adding such asserts causes the tests to fail.